src/20240322_AaveV2Polygon_ReserveFactorUpdates/ReserveFactorUpdates.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "ReserveFactorUpdates"
+author: "TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937/21"
+---
+
+## Simple Summary
+
+This AIP is a continuation of proposal 54 on Governance v3 and increases the Reserve Factor (RF) for assets on Polygon v2 by 5.00%, up to a maximum of 99.99%.
+
+## Motivation
+
+This AIP will reduce deposit yield for assets on Polygon v2 by increasing the RF. With this upgrade being passed, users will be further encouraged to migrate from Polygon v2 to v3.
+
+Increasing the RF routes a larger portion of the interest paid by users to Aave DAO's Treasury. User's funds are not at risk of liquidation and the borrowing rate remains unchanged.
+
+Of the assets with an RF set at 99.99%, there is no change. All other asset reserves will have the RF increased by up to 5%.
+
+## Specification
+
+The following parameters are to be updated as follows:
+
+| Asset | Reserve Factor |
+| ----- | -------------- |
+| DAI   | 91.00%         |
+| USDC  | 93.00%         |
+| USDT  | 92.00%         |
+
+## References
+
+- Implementation: [AaveV2Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240322_AaveV2Polygon_ReserveFactorUpdates/AaveV2Polygon_ReserveFactorUpdates_20240322.sol)
+- Tests: [AaveV2Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240322_AaveV2Polygon_ReserveFactorUpdates/AaveV2Polygon_ReserveFactorUpdates_20240322.t.sol)
+- [Snapshot](TODO)
```

Comment:
Why do you have TODO here?

Reply:
resolved in [f93dfa4](https://github.com/bgd-labs/aave-proposals-v3/pull/284/commits/f93dfa4c8f19d7eb03a2c8f36983d2a659a0a896). There is no snapshot to my knowledge and omitting this line is consistent with the previous Polygon RF updates